docs/snapshots.md

```diff
@@ -0,0 +1,54 @@
+# Index Snapshots
+
+This document describes index snapshots, what they are and how they can be created or updated.
+
+**Table of Contents**
+
+- [What Are Index Snapshots](#what-are-index-snapshots)
+- [Creating a Snapshot](#creating-a-snapshot)
+- [Updating a Snapshot](#updating-a-snapshot)
+    1. [New Index](#new-index)
+    2. [New Snapshot](#new-snapshot)
+
+## What Are Index Snapshots
+
+Index snapshots are images of the DPS index at a certain point in time.
+These images can be used to create or restore the content of an index in order to operate on a certain indexed data.
+Index snapshots can be used to create a starting point for tests, so that tests have actual blocks, accounts, transactions and other information to operate on.
+
+At a low level, snapshots are created using the [badger](https://github.com/dgraph-io/badger) backup functionality.
+Technical documentation can be found [here](https://pkg.go.dev/github.com/dgraph-io/badger/v3#DB.Backup).
```

Comment:
At the time when I was writing this, their `v2` go doc page was broken, so I linked `v3` instead, since the relevant part was the same.

Now they've fixed it, so we can correctly link https://pkg.go.dev/github.com/dgraph-io/badger/v2#DB.Backup now.